__tests__/app.test.js

```diff
@@ -298,3 +298,106 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/article_id", () => {
+  it("Patch 200: responds with the updated article where the votes count will have been altered correctly", () => {
+    const votes = { inc_votes: 13 };
+    return request(app)
+      .patch("/api/articles/7")
```

Comment:
A good thing to do here might be store the 7 in a variable something like `const validId = 7` similarly for invalid ones just to add an extra layer of semantics to your code